events/wc_society_events.txt

```diff
@@ -173,3 +173,303 @@ character_event = {
+    	trigger = {
+    		NOT = {
+    			has_artifact = whisphers_stewardship_1
+				has_artifact = whisphers_stewardship_2
+				has_artifact = whisphers_stewardship_3
+				has_artifact = whisphers_stewardship_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whisphers_stewardship
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
+    	}
+    }
+
+    option = { #intrigue
+    	name = EVTOPTC_WCCSO.10201_tester_info_intrigue
+    	trigger = {
+    		NOT = {
+    			has_artifact = whisphers_intrigue_1
+				has_artifact = whisphers_intrigue_2
+				has_artifact = whisphers_intrigue_3
+				has_artifact = whisphers_intrigue_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whisphers_intrigue
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
+    	}
+    }
+	option = { #learning
+    	name = EVTOPTD_WCCSO.10201_tester_info_learning
+    	trigger = {
+    		NOT = {
+    			has_artifact = whisphers_learning_1
+				has_artifact = whisphers_learning_2
+				has_artifact = whisphers_learning_3
+				has_artifact = whisphers_learning_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whisphers_learning
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
+    	}
+    }
+}
+
+character_event = {
+	id = WCCSO.10202
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_martial }
+		text = EVTDESC_WCCSO.10202_martial
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_stewardship }
+		text = EVTDESC_WCCSO.10202_stewardship
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_intrigue }
+		text = EVTDESC_WCCSO.10202_intrigue
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_learning }
+		text = EVTDESC_WCCSO.10202_learning
+	}
+	picture = GFX_evt_secluded_in_prayer
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+	option = {
+		name = EVTOPTA_WCCSO.10202
+    	hidden_effect = {
+    		character_event = { id = WCCSO.10203 days = 90 random = 60 }
+    	}
+	}
+}
+
+character_event = {
+	id = WCCSO.10203
+	desc = EVTDESC_WCCSO.10203
+	picture = GFX_evt_comet
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+	option = {
+		name = EVTOPTA_WCCSO.10203
+    	if = {
+			limit = {
+				has_character_flag = writing_whisphers_martial 
+			}
+			if = {
+				limit = {
+					NOT = { learning = 12 }
+				}
+				piety = 50
+				add_artifact = whisphers_martial_1
+			}
+			if = {
+				limit = {
+					AND = {
+						learning = 12
+						NOT = { learning = 18 }
+					}
+				}
+				piety = 100
+				add_artifact = whisphers_martial_2
+			}
+			if = {
+				limit = {
+					AND = {
+						learning = 18
+						NOT = { learning = 25 }
+					}
+				}
+				piety = 150
+				add_artifact = whisphers_martial_3
+			}
+			if = {
+				limit = {
+					learning = 25
+				}
+				piety = 200
+				add_artifact = whisphers_martial_4
+			}
+		}
```

Comment:
Weird. Having Learning doesn't mean you're good at martial. The same question to other books.

Reply:
high learning = better hearing of whisphers = better martial

Reply:
Gameplay-wise, it doesn't work well. You're learning guy, most likely you don't have martial, so martial book doesn't do much.

Reply:
i mean 4 martial as a learning guy can make a big difference

Reply:
i can base it on piety if you want

Reply:
Cho'gall is a good learning character. With this book, he will have 11 martial. So, randomly summoned general will be better. Plus, you don't risk your life.

Reply:
yes but you get more troops

Reply:
> i can base it on piety if you want

It works.